android/src/main/java/io/invertase/firebase/notifications/RNFirebaseNotifications.java

```diff
@@ -284,6 +289,7 @@ private WritableMap parseRemoteMessage(RemoteMessage message) {
     }
     if (notification.getTag() != null) {
       androidMap.putString("group", notification.getTag());
```

Comment:
Now that we have the `tag` property, I don't think we should also set that as the `group` as they are two separate things.  Are you able to remove the `group`?

Reply:
Removing the group could be a breaking change for some people, because of that I'm not sure if we should do that.